test/modules/jupyter-widget/widget-tooltip.spec.js

```diff
@@ -0,0 +1,143 @@
+// eslint-disable-next-line
+import test from 'tape-catch';
+
+import {getModule} from './utils.spec';
+
+const TOOLTIP_LIB = '@deck.gl/jupyter-widget/widget-tooltip';
+
+const pickedInfo = {object: {elevationValue: 10, position: [0, 0]}, x: 0, y: 0, picked: true};
+
+// eslint-disable-next-line
+const TOOLTIP_HTML = {
+  html:
+    '<div style="display: flex; flex-direction: row; justify-content: space-between; align-items: stretch;"><div class="header" style="font-weight: 700; margin-right: 10px; flex: 1;"></div><div class="value" style="flex: 0 0 auto; max-width: 250px; overflow: hidden; white-space: nowrap; text-overflow: ellipsis;"></div></div>',
+  style: {
+    fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+    display: 'flex',
+    flex: 'wrap',
+    maxWidth: '500px',
+    flexDirection: 'column',
+    zIndex: 2
+  }
+};
+
+function safelyGetModule(t) {
+  // Skip tests if in headless browser environment
+  let module;
+  try {
+    module = getModule(TOOLTIP_LIB);
+  } catch (err) {
+    return null;
```

Comment:
You should never use try...catch without logging and/or documentation. If there is a bug in your code that causes crash, your tests here will be useless in catching it.

- What exactly in the tooltip module breaks in the Node environment?
- Why do you need nested try...catch in this util and `getModule`? What error could be generated?